rust/cubesql/cubesql/src/compile/rewrite/analysis.rs

```diff
@@ -55,6 +55,16 @@ pub enum OriginalExpr {
     List(Vec<Expr>),
 }
 
+#[derive(Clone, Debug)]
+pub struct MemberNamesToExpr {
+    /// List of MemberNameToExpr.
+    pub list: Vec<MemberNameToExpr>,
+    /// Results of lookup_member_by_column_name represented as indexes into `list`.
+    // Note that using Vec<(String, usize)> had nearly identical performance the last time that was
+    // benchmarked.
+    pub cached_lookups: HashMap<String, usize>,
```

Comment:
Is it a small hashmap with less than 30 elements? Then you can use `minimap`.
Otherwise, it's better to use `hashbrowns`.

Reply:
What is minimap?  I can't find anything about it.

The std HashMap is now the same as the one that was in hashbrown, as of Rust 1.36.  But using hashbrown here will let us use try_insert without a feature flag.

Reply:
This now uses hashbrown::HashMap, but note that all the other code in the cubesql folder uses std::collections::HashMap.